src/python/WMCore/WMSpec/StdSpecs/Express.py

```diff
@@ -181,7 +181,8 @@ def buildWorkload(self):
                 alcaSkimTask = expressTask.addTask("%sAlcaSkim%s" % (expressTask.name(), expressOutLabel))
 
                 alcaSkimTask.setInputReference(expressTask.getStep(expressRecoStepName),
-                                               outputModule=expressOutLabel)
+                                               outputModule=expressOutLabel,
+                                               dataTier="ALCARECO")
```

Comment:
Instead of hard-coding the datatier, don't you have it in another variable?

Reply:
We are in a conditional "if" code segment here that checks for the dataTier to be ALCARECO. Seemed redundant to put the variable again.

Reply:
If you ever change this tier name, then you have one more place to update, so chances to miss it are higher :-)